masu/external/downloader/aws/aws_report_downloader.py

```diff
@@ -69,7 +69,8 @@ def __init__(self, customer_name, auth_credential, bucket, report_name=None, **k
         self.customer_name = customer_name.replace(' ', '_')
 
         LOG.debug('Connecting to AWS...')
-        session = utils.get_assume_role_session(auth_credential, 'MasuDownloaderSession')
+        session = utils.get_assume_role_session(utils.AwsArn(auth_credential),
```

Comment:
Could we do something like this:
```
try:
    self._rolearn = utils.AwsArn(auth_credential)
except SyntaxError:
    raise AWSDownloaderError('Incorrect RoleARN format')
session = utils.get_assume_role_session(self._rolearn, 'MasuDownloaderSession')
```
That way when it blows up, the backtrace will be very clear about what went wrong.

Reply:
ok no biggie.